app/src/main/res/layout/fragment_details.xml

```diff
@@ -4,34 +4,38 @@
         xmlns:app="http://schemas.android.com/apk/res-auto"
         xmlns:tools="http://schemas.android.com/tools"
         tools:context=".Fragments.DetailsFragment">
-
+    <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
     <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:gravity="center"
             android:padding="30dp">
 
+        <TextView
+                android:id="@+id/txt1"
```

Comment:
change the id to profileTxt.

Reply:
Since you are making a change, also change the id of the button from editFab to editButton.

Reply:
@rob729 I have changed the ids for both, please check

Reply:
@rob729 I have made changes that were asked. Please have a look